h/harfbuzz/manifest.x86_64.jsonc

```diff
@@ -20,21 +20,20 @@
 				"soname(libgobject-2.0.so.0(x86_64))",
 				"soname(libgraphite2.so.3(x86_64))",
 				"soname(libicuuc.so.75(x86_64))",
-				"soname(libm.so.6(x86_64))",
-				"soname(libstdc++.so.6(x86_64))"
```

Comment:
It's very interesting that this disappears but that it didn't link against libc++ instead. I don't think it's an issue, just surprised to see that.